src/components/Article.js

```diff
@@ -35,6 +37,10 @@ const ArticleSection = ({
     },
     reference: ({ referenceId }) => {
       showReferencePreview({ reference: references[referenceId], lang })
+    },
+    goto: ({ anchor }) => {
```

Comment:
I don't think `goto` is appropriate. The other keys represent what the link is. Maybe `section` would be more clear.